x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.test.ts

```diff
@@ -545,6 +530,28 @@ describe('RelevanceTuningLogic', () => {
         expect(flashAPIErrors).toHaveBeenCalledWith('error');
         expect(RelevanceTuningLogic.actions.onSearchSettingsError).toHaveBeenCalled();
       });
+
+      it('will re-fetch the current engine after settings are updated if there were invalid boosts', async () => {
+        mockEngineValues.engine.invalidBoosts = true;
+        mount({});
+        http.put.mockReturnValueOnce(Promise.resolve(searchSettings));
+
+        RelevanceTuningLogic.actions.updateSearchSettings();
+        await nextTick();
+
+        expect(mockEngineActions.initializeEngine).toHaveBeenCalled();
+      });
+
+      it('will re-fetch the current engine after settings are updated if there were unconfirmed search fieldds', async () => {
```

Comment:
damnit, only just noticed this, sorry! optional though, feel free to fix in a future PR instead if you'd rather save time on CI cycles

```suggestion
      it('will re-fetch the current engine after settings are updated if there were unconfirmed search fields', async () => {
```